src/Couchbase.Lite.Shared/API/Sync/Replicator.cs

```diff
@@ -219,7 +219,10 @@ public void ResetCheckpoint()
         /// <summary>
         /// Starts the replication
         /// </summary>
-        public void Start()
+        /// <param name="reset">Resets the local checkpoint of the replicator, meaning that it will read all changes since the beginning
+        /// of time from the remote database.
+        /// </param>
+        public void Start(bool reset)
```

Comment:
This needs to be an overload, not a replacement, otherwise the API will break.

Reply:
Done